src/localisation-service.ts

```diff
@@ -127,6 +127,17 @@ export function formatDateLong(value: Date): string {
  * @returns
  */
 export function formatTime(value: Date): string {
+  const iosVersion = Number.parseFloat(navigator.userAgent.split(' ')[5].replace("_", "."))
```

Comment:
What will happen if the userAgent is not for a iOS device?

Reply:
This PR can be closed, used wrong ticket number, have started a new PR at https://github.com/oneblink/apps/pull/200